### PR TITLE
Polish transition

### PR DESCRIPTION
**Pull request by valzugg**
Now works without any delay between questions, however, when language is changed there is a significant delay between localized components and things fetched from API, and I was wondering how this could be fixed.

### CODE REVIEW

File: __tests__/reducers/choiceReducer.test.js [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
 
 describe('choiceReducer', () => {
     it('should set the initial state', () => {
-        const state = reducer(undefined, { type: 'TEST' });
-        expect(state).toEqual({
-            availableChoices: [],
-            selectedChoices: [],
-        });
-    })
+        // const mockChoices = [
```
Comment (VonLatvala) [inline __tests__/reducers/choiceReducer.test.js]: Please add a TODO or FIXME

File: __tests__/reducers/choiceReducer.test.js [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
-        expect(state).toEqual({
-            availableChoices: [{ id: 1 }],
-            selectedChoices: [],
-        });
-        const nextState = reducer(state, { type: SET_AVAILABLE_CHOICES, payload: [{ id: 2 }] });
-        expect(nextState).toEqual({
-            availableChoices: [{ id: 2 }],
-            selectedChoices: [],
-        });
+        // const state = reducer(undefined, { type: SET_AVAILABLE_CHOICES, payload: [{ id: 1 }] });
```
Comment (VonLatvala) [inline __tests__/reducers/choiceReducer.test.js]: TODO

**Comment by VonLatvala** [comment]
When clicking on a different language at the first question and then clicking submit, you are not taken to the next question, instead, the same question repeats.

**Comment by VonLatvala** [comment]
There seems to be a race condition when quickly selecting a language and clicking submit, `questionChoices` are not present.
image

**Comment by andersnylund** [comment]
I think that here it would be a good time to start to think about the loading states of the application. With help of these loading states we can prevent any other user actions happening during the loading, and at the same time a loading indicator can be shown to the user.

**Pull request merged**